src/Console/Starter.php

```diff
@@ -86,7 +86,7 @@ public function handle()
             if ($this->confirm('Would you like to run the migration?')) {
                 $this->comment('Running command: cd '.base_path().' && composer dump ...');
 
-                exec('cd '.base_path().' && composer dump');
+                exec(sprintf('cd %s && composer dump', escapeshellarg(base_path()));
```

Comment:
In truth this could also be 

```
exec('cd '.escapeshellarg(base_path()).' && composer dump');
```
but I thought the `sprintf` might look cleaner